### PR TITLE
avoid splitting script into feature files/scripts

### PR DESCRIPTION
**Pull request by skshetry**
Reverts https://github.com/iterative/dvcx/pull/1500.

I don't think this is needed after we started using cloudpickle for parallel udfs since https://github.com/iterative/datachain/pull/65.

Part of #360 .

### CODE REVIEW

**Comment by skshetry** [comment]
ping @dtulga.

**Review by amritghimire — APPROVED** [review]
Lets wait for Studio CI to be green before merging this

**Comment by skshetry** [comment]
All of the tests passed in Studio. See above referenced link.

**Pull request merged**
